### DISCUSSION TITLE
error Error: Invalid token

### DISCUSSION THREAD

**Discussion by seguelador** [strapi/strapi#7462] [Legacy Topics]
Hi

I'm working with Strapi and running through an error that can't solve. 

So, I basically deployed my Strapi app and when trying to access to /admin this is what I get:

- /users-permissions/init request returns a 401 Unauthorized.
- Console log says: 

`

[2020-08-13T21:19:53.704Z] error Error: Invalid token.
--
  | at /home/app/node_modules/strapi-plugin-users-permissions/services/Jwt.js:64:27
  | at Object.module.exports [as verify] (/home/app/node_modules/jsonwebtoken/verify.js:53:12)
  | at /home/app/node_modules/strapi-plugin-users-permissions/services/Jwt.js:56:11
  | at new Promise (<anonymous>)
  | at Object.verify (/home/app/node_modules/strapi-plugin-users-permissions/services/Jwt.js:55:12)
  | at Object.getToken (/home/app/node_modules/strapi-plugin-users-permissions/services/Jwt.js:40:17)
  | at module.exports (/home/app/node_modules/strapi-plugin-users-permissions/config/policies/permissions.js:10:22)
  | at dispatch (/home/app/node_modules/koa-compose/index.js:42:32)
  | at /home/app/node_modules/strapi-utils/lib/policy.js:118:13
  | at dispatch (/home/app/node_modules/koa-compose/index.js:42:32)
  | at /home/app/node_modules/koa-compose/index.js:34:12
  | at dispatch (/home/app/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
  | at next (/home/app/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
  | at /home/app/node_modules/koa-router/lib/router.js:346:16
  | at dispatch (/home/app/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
  | at /home/app/node_modules/koa-router/node_modules/koa-compose/index.js:36:12

`

am I missing any configurations? Any help would be appreciated

**Comment by derrickmehaffy** [comment]
Do you have any extensions to the users-permission plugin like customizing one of the policies?

**Reply by seguelador** [reply]
No, I don't. I activated graphql by installing it using the strapi dashboard (plugins) and running `npm run strapi install graphql` only

**Reply by seguelador** [reply]
I am also using strapi docker (strapi/strapi image)

**Comment by seguelador** [comment]
I have found the cause of the issue. At work, we are using a proxy-reverse strategy for security measures, due to the latter the token key was not being passed through the headers. 
Thanks :D
